src/systemdhook.c

```diff
@@ -822,8 +822,14 @@ int main(int argc, char *argv[])
 		snprintf(config_file_name, PATH_MAX, "%s/config.json", YAJL_GET_STRING(v_bundle_path));
 		fp = fopen(config_file_name, "r");
 	} else {
-		char msg[] = "bundlePath not found in state";
+		/****
+		* On Docker versions prior to 1.12, bundlePath will not
+		* be provided.  Let's exit quietly if not found.
+		****/
+		char msg[] = "bundlePath not found";
 		snprintf(config_file_name, PATH_MAX, "%s", msg);
+		pr_pinfo("Failed reading state data: %s.  Generally this indicates Docker versions prior to 1.12 are installed.", config_file_name);
+		return EXIT_SUCCESS;
 	}
```

Comment:
Nit: Why not just pr_pinfo the message rather then using config_file_name at all.